content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,382 @@
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+e-Invoicing
+-----------
 
-To receive invoices and notifications from third parties, you need to inform the FatturaPA service
-that Odoo is the allowed party to process files for you. To do so, you must setup Odoo's *Codice
-Destinatario* on the FatturaPA portal. The *Codice Destinatario* is ``K95IV18``.
+The :abbr:`SdI (Sistema di Interscambio)` is the electronic invoicing system used in Italy. It
+enables transmission and reception of electronic billing in XML format, as well as validation and
+sending of XML e-Invoices from businesses to third parties.
 
-#. Go to https://ivaservizi.agenziaentrate.gov.it/portale/ and authenticate.
-#. Go to section :menuselection:`Fatture e Corrispettivi`.
-#. Set the user as Legal Party for the VAT number you wish to configure the electronic adress.
+To receive invoices and notifications, the **FatturaPA** service must be notified of Odoo's
+authorization to process files for you. To do so, you must set up Odoo's :guilabel:`Codice
+Destinatario` on the FatturaPA portal.
+
+#. Go to https://ivaservizi.agenziaentrate.gov.it/portale/ and authenticate;
+#. Go to section :menuselection:`Fatture e Corrispettivi`;
+#. Set the user as Legal Party for the VAT number you wish to configure the electronic address;
 #. In :menuselection:`Servizi Disponibili --> Fatturazione Elettronica --> Registrazione
-   dell’indirizzo telematico dove ricevere tutte le fatture elettroniche`, input Odoo's Codice
-   Destinatario (**K95IV18**), then confirm.
+   dell’indirizzo telematico dove ricevere tutte le fatture elettroniche`, insert Odoo's
+   :guilabel:`Codice Destinatario` `K95IV18`, and confirm.
+
+.. seealso::
+   :doc:`../../receivables/customer_invoices/electronic_invoicing`
 
-Give Odoo permission to process files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+File processing authorization (Odoo)
+------------------------------------
 
 Since the files are transmitted through Odoo's server before being sent to SDICoop or received by
-your database, you need to authorize Odoo to process your files from your database.
+your database, you need to authorize Odoo to process your files from your database. To do so, go to
+:menuselection:`Accounting --> Configuration --> Settings --> Electronic Document Invoicing`.
 
-To do this, go to :menuselection:`Accounting --> Settings --> Electronic Invoicing` and click on
-*Register*.
+There are **three** modes available:
 
-.. image:: italy/register.png
-   :align: center
-   :alt: Register button to allow Odoo to process invoices.
+- :guilabel:`Demo` mode simulates an environment in which invoices are sent to the government. In
+  this mode, invoices need to be *manually* downloaded as XML files and uploaded to the **Agenzia
+  delle Entrate**'s website;
+- :guilabel:`Test (experimental)` mode sends invoices to a non-production (i.e., test) service.
+  Invoices are sent from Odoo to selected *fake* destinations of the **Pubblica Amministrazione**'s
```

Comment:
@lordkrandel Is the Pubblica Amministrazione the same as the Agenzia delle Entrate ?